### ISSUE TITLE
x86: Using an x86 image, not all the interfaces are picked up & cannot be set in UI

### ISSUE THREAD

**Issue by maurerle** [freifunk-gluon/gluon#3129]
There are two issues with unconfigured nodes:

## 1. when I have a device with three NICs like this

```
root@ffac-bc24112bd4e5:/# ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel master br-setup state UP group defa0
    link/ether bc:24:11:2b:d4:e5 brd ff:ff:ff:ff:ff:ff
3: eth1: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether bc:24:11:3f:82:d0 brd ff:ff:ff:ff:ff:ff
4: dummy0: <BROADCAST,NOARP> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether e2:28:dc:52:9f:bd brd ff:ff:ff:ff:ff:ff
5: eth2: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether bc:24:11:a1:d4:6a brd ff:ff:ff:ff:ff:ff
6: br-setup: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether bc:24:11:2b:d4:e5 brd ff:ff:ff:ff:ff:ff
    inet 192.168.1.1/24 brd 192.168.1.255 scope global br-setup
       valid_lft forever preferred_lft forever
    inet6 fe80::be24:11ff:fe2b:d4e5/64 scope link 
       valid_lft forever preferred_lft forever
```

The default configuration does not include eth2 at all:

```
root@ffac-bc24112bd4e5:/# uci show network | grep eth
network.wan.ifname='eth1'
network.client.ifname='eth0' 'local-port' 'bat0'
network.local_node_dev.type='veth'
```

Eth2 is just not used at all for such machines.
From my experience, this has always been the case for x86 and it would be very helpful to automatically discover other nics too.

## 2. Furthermore it is not possible to set a function for the third NIC in the Network web ui

![image](https://github.com/freifunk-gluon/gluon/assets/25026204/961d2279-b651-46a5-a360-5ebfb6082252)

This second issue also appears on some devices which have something like
`lan_ifname, wan_ifname = 'lan2 lan3 lan4', 'lan1'`
in here:
https://github.com/freifunk-gluon/gluon/blob/d517e0ff7c065fde70e098f2659a828c02f13ce2/package/gluon-core/luasrc/lib/gluon/upgrade/020-interfaces

And probably others..?
While it is technically (and through UCI) possible to use for example port1 as WAN, port2 as LAN and port3/4 as mesh - this option is not possible in the checkboxes..

Eventually, it is not easily possible to improve this, but I always wondered why this is the case.

**Comment by maurerle** [comment]
This might be implemented by #2907 in the future which is blocked by #2296

**Comment by maurerle** [comment]
I'll close this issue, as there is nothing going forward in the future for this case, and it is already covered by the other issues.
Hopefully, #2296 will get some traction in the future
